src/pybind/pyembed.cpp

```diff
@@ -92,7 +92,8 @@ void EmbeddedPythonLoader::load_libraries() {
                          CMakeInfo::SHARED_LIBRARY_SUFFIX);
         throw std::runtime_error("NMODLHOME doesn't have lib/libpywrapper library");
     }
-    pybind_wrapper_handle = dlopen(pybind_wraplib_env.c_str(), dlopen_opts);
+    std::string env_str = pybind_wraplib_env.string();
+    pybind_wrapper_handle = dlopen(env_str.c_str(), dlopen_opts);
```

Comment:
Do you know why this is required? It's not obvious from cppreference.

Reply:
Coercing the path directly to `char *` doesn't work on Windows, I think the underlying path structure differs, it needs some explicit conversion.

Reply:
```
[build] C:\Users\matze\Documents\GitHub\nmodl\src\pybind\pyembed.cpp(95,75): error C2664: 'void *dlopen(const char *,int)': cannot convert argument 1 from 'const std::filesystem::path::value_type *' to 'const char *' [C:\Users\matze\Documents\GitHub\nmodl\build\src\pybind\pyembed.vcxproj]
[build] C:\Users\matze\Documents\GitHub\nmodl\src\pybind\pyembed.cpp(95,60): message : Types pointed to are unrelated; conversion requires reinterpret_cast, C-style cast or function-style cast [C:\Users\matze\Documents\GitHub\nmodl\build\src\pybind\pyembed.vcxproj]
[build] C:\Users\matze\Documents\GitHub\nmodl\ext\dlfcn-win32\src\dlfcn.h(76,20): message : see declaration of 'dlopen' [C:\Users\matze\Documents\GitHub\nmodl\build\src\pybind\pyembed.vcxproj]
```

My error... maybe I need to upgrade my local VS. More modern versions allegedly also allow longer string literals.

Reply:
Okay, so this is what I overlooked:
> `value_type` 	character type used by the native encoding of the filesystem: `char` on POSIX, `wchar_t` on Windows